CONTRIBUTING.md

```diff
@@ -420,6 +427,28 @@ $ ./node ./test/parallel/test-stream2-transform.js
 Remember to recompile with `make -j4` in between test runs if you change code in
 the `lib` or `src` directories.
 
+##### Test Coverage
+
+It's good practice to ensure any code you add or change is covered by tests.
+You can do so by running the test suite with coverage enabled:
+
+```text
+$ ./configure --coverage && make coverage
+```
+
+A detailed coverage report will be output to `coverage/index.html` for
+JavaScript coverage and to `coverage/cxxcoverage.html` for C++ coverage.
+
+To collect coverage for a subset of tests you can set the `CI_JS_SUITES` and
+`CI_NATIVE_SUITES` variables:
+
+```text
+$ CI_JS_SUITES=child-process CI_NATIVE_SUITES= make coverage
+```
+
+The above command executes tests for the `child-process` subsystem and
+outputs the resulting coverage report.
```

Comment:
Sorry to keep nits coming on this, but one more: People new to using `make coverage` will probably want to know that when you're done, you should run `make coverage-clean` (and probably `./configure` for good measure) to get things back to the non-modified-for-coverage state. Maybe something like this?:

> Running coverage tests will create many directories and files. To clean up these new directories and files, run `make coverage-clean`. To avoid accidentally using the executable that has been modified for coverage reports, you may wish to also run `./configure && make -j4`.

Reply:
No worries, appreciate the thorough review. I just pushed your recommendations, let me know what you think and I'll rebse.